EIPS/eip-symbol.md

```diff
@@ -0,0 +1,61 @@
+---
+eip: <to be assigned>
+title: Create `eth_symbol` method for JSON-RPC
+author: Peter Grassberger (@PeterTheOne)
+discussions-to: https://github.com/ethereum/EIPs/issues/3012
+status: Draft
+type: Standards Track
+category : Interface
+created: 2020-09-30
+requires (*optional): <EIP number(s)>
+replaces (*optional): <EIP number(s)>
+---
+
+## Simple Summary
+Add `eth_symbol` method to the JSON-RPC that returns the symbol of the native coin of the network.
+
+## Abstract
+Wallets (Metamask, Mobile Wallets, Web Wallets, etc.) that deal with multiple networks need some basic information for every blockchain that they connect to. One of those things is the symbol of the native coin of the network. Instead of requiring the user to research and manually add the symbol it could be provided to the wallet via this proposed JSON-RPC endpoint and used automatically.
```

Comment:
This content should be moved to `## Motivation` section.  Abstract should be a human readable and terse version of the specification.  In this case, it may not be much longer than the Simple Summary since this EIP is nice and simple.  Maybe mention that the result is a string?

Reply:
done